app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/login/LoginScreenHeader.kt

```diff
@@ -2,34 +2,69 @@ package ch.epfl.cs311.wanderwave.ui.components.login
 
 import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.FloatingActionButton
 import androidx.compose.material3.Icon
 import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
 import ch.epfl.cs311.wanderwave.R
+import ch.epfl.cs311.wanderwave.navigation.NavigationActions
+import ch.epfl.cs311.wanderwave.navigation.Route
+import ch.epfl.cs311.wanderwave.ui.components.utils.lerp
+import ch.epfl.cs311.wanderwave.ui.theme.placeholderColor
 
 @Composable
-fun LoginScreenHeader(modifier: Modifier) {
-
-  Column(
-      horizontalAlignment = Alignment.CenterHorizontally,
-      verticalArrangement = Arrangement.Center,
-      modifier = modifier.fillMaxWidth().padding(16.dp)) {
-        Icon(
-            painter = painterResource(id = R.drawable.app_logo),
-            contentDescription = "app logo",
-            tint = MaterialTheme.colorScheme.primary,
-            modifier =
-                Modifier.height(50.dp)
-                    .fillMaxWidth()
-                    .padding(start = 50.dp, end = 50.dp)
-                    .testTag("appLogo"))
+fun LoginScreenHeader(navigationActions: NavigationActions, modifier: Modifier) {
+  val startColor = MaterialTheme.colorScheme.primary
+  val endColor = placeholderColor
+  val colorSpots = List(10) { i -> lerp(startColor, endColor, i / 9f) }
+  FloatingActionButton(
+      onClick = { navigationActions.navigateTo(Route.ABOUT) },
```

Comment:
I don't think the Login Screen is the best place to link to the about screen. From my experience, it's usually in the settings or in the users profile where it's linked. The login screen will be seen for a very short time before pressing the login button by most users.
If we leave it here, agian, FloatingActionButton seems weird.

Reply:
Agree that simpler design is better.